Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -75,17 +113,17 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
-    new_version = unless (new_version = args.version).nil?
-      raise UsageError, "`--version` must not be empty." if new_version.blank?
-
-      new_version = :latest if ["latest", ":latest"].include?(new_version)
-      Cask::DSL::Version.new(new_version)
-    end
+    new_version = Homebrew::NewVersion.new(
+      general: args.version,
+      intel:   args.version_intel,
+      arm:     args.version_arm,
+    )
 
-    new_hash = unless (new_hash = args.sha256).nil?
+    new_hash = args.sha256
+    if new_hash.present?
```

Comment:
Checking for `present?` here prevents the `blank?` check below from ever being reached.

Reply:
Thanks, should be fixed.